src/easylink/pipeline_schema_constants/development.py

```diff
@@ -1,24 +1,89 @@
 from easylink.graph_components import Edge, InputSlot, OutputSlot, SlotMapping
-from easylink.step import BasicStep, HierarchicalStep, InputSlot, IOStep, LoopStep
+from easylink.step import (
+    BasicStep,
+    HierarchicalStep,
+    InputSlot,
+    IOStep,
+    LoopStep,
+    ParallelStep,
+)
 from easylink.utilities.validation_utils import validate_input_file_dummy
 
 NODES = [
-    IOStep(step_name="input_data", input_slots=[], output_slots=[OutputSlot("file1")]),
-    BasicStep(
+    IOStep(step_name="input_data", input_slots=[], output_slots=[OutputSlot("all")]),
+    ParallelStep(
```

Comment:
nit: should it be MultipleStep to keep w/ the naming conventions?

Reply:
@zmbc made an offhand comment that we maybe shouldn't actually call these 'multiples' in the actual implementation, but I'm happy to call them whatever.

Reply:
👍 I'd like to update the milestones doc to also say "parallel" since I think that is more informative.